### PR TITLE
Vendors API

### PR DESCRIPTION
**Pull request by maescomua**

### CODE REVIEW

File: spec/swagger.yaml [hunk 1]
```diff
@@ -183,6 +183,11 @@ tags:
   - name: Tags
     description: |-
       Tags can help you manage your strings. The Tags API allows you to add/remove string tags, and search for tags in a project.
+
+  - name: Vendors
+    description: |-
+      Vendors API is to be used by LSPs to view work available in Smartling.
```
Comment (hchaps) [inline spec/swagger.yaml]: The Vendors API is for Language Service Providers to view work that's available in Smartling.

File: spec/swagger.yaml [hunk 2]
```diff
@@ -10031,3 +10062,134 @@ definitions:
...
+          type: object
+          required:
+            - lastUpdatedDate
+            - agency
+            - translationJobs
+          properties:
+            lastUpdatedDate:
+              type: string
+              format: date-time
+              description: Date when data for the api response was retrieved.
```
Comment (hchaps) [inline spec/swagger.yaml]: API

File: spec/swagger.yaml [hunk 3]
```diff
@@ -10031,3 +10062,134 @@ definitions:
...
+    properties:
+      wordCount:
+        type: integer
+      unassignedWordCount:
+        type: integer
+      workflowStepName:
+        type: string
+      workflowStepOrder:
+        type: integer
+        description: Position of a workflow step in workflow
```
Comment (hchaps) [inline spec/swagger.yaml]: A bit unclear...?

File: spec/swagger.yaml (hunk 2 above)
Comment (jones-smartling) [reply spec/swagger.yaml]: The description doesnt sound right, it's more when the response returned as last calculated.

File: spec/swagger.yaml [hunk 4]
```diff
@@ -10031,3 +10062,144 @@ definitions:
...
+  VendorsAPIJobsListAgencyItemProgressDetailsItemWorkflowsItemTargetLocalesItemWorkflowStepsItem:
+    type: object
+    properties:
+      wordCount:
+        type: integer
+      unassignedWordCount:
+        type: integer
+      workflowStepName:
+        type: string
+      workflowStepOrder:
```
Comment (jones-smartling) [inline spec/swagger.yaml]: based on upcoming changes with workflows, workflowStepOrder may be something we want to hold off on. @klegins-smartling?

File: spec/swagger.yaml [hunk 5]
```diff
@@ -10031,3 +10062,144 @@ definitions:
...
+    properties:
+      translationJobAuthorizedDate:
+        type: string
+        format: date-time
+      translationJobDueDate:
+        type: string
+        format: date-time
+      translationJobName:
+        type: string
+      translationJobUidtranslationJobUid:
```
Comment (jones-smartling) [inline spec/swagger.yaml]: this looks incorrect

File: spec/swagger.yaml [hunk 6]
```diff
@@ -6553,6 +6558,32 @@ paths:
...
+      responses:
+        200:
+          description: OK
+          schema:
+            $ref: '#/definitions/VendorsAPIJobsListSuccessResponse'
+        400: *Error400ResponseDefinition
+        401: *Error401ResponseDefinition
+        429: *Error429ResponseDefinition
+        500: *Error500ResponseDefinition
+      x-code-samples:
```
Comment (jones-smartling) [inline spec/swagger.yaml]: can a sample response be added?

**Pull request merged**